.github/workflows/workflow-trigger-packaging.yml

```diff
@@ -0,0 +1,65 @@
+name: Trigger Packaging
+
+on:
+  workflow_call:
+    inputs:
+      create_release:
+        description: The packaging workflow will create a draft release in the packaging repository.
```

Comment:
This only triggers packaging during a release build?

Reply:
We don't trigger packaging at all for releases right now. I'd like to add that in a separate PR.

Reply:
oh nvm, we seem to be using this during a dev build